content/pl/docs/concepts/overview/_index.md

```diff
@@ -1,7 +1,168 @@
+[piętnastoletniego doświadczenia Google w uruchamianiu wielkoskalowych serwisów](/blog/2015/04/borg-predecessor-to-kubernetes/)
+i łączy je z najlepszymi pomysłami i praktykami wypracowanymi przez społeczność.
+
+## Trochę historii
+
+Aby zrozumieć, dlaczego Kubernetes stał się taki przydatny, cofnijmy sie trochę w czasie.
+
+![Jak zmieniały sie metody wdrożeń](/images/docs/Container_Evolution.svg)
+
+**Era wdrożeń tradycyjnych:**
+Na początku aplikacje uruchamiane były na fizycznych serwerach. Nie było możliwości separowania zasobów poszczególnych aplikacji,
+co prowadziło do problemów z alokacją zasobów.
+Przykładowo, kiedy wiele aplikacji jest uruchomionych na jednym fizycznym serwerze,
+część tych aplikacji może zużyć większość dostępnych zasobów, powodując spowolnienie działania innych.
+Rozwiązaniem tego problemu mogło być uruchamianie każdej aplikacji na osobnej maszynie.
+Niestety, takie podejście ograniczało skalowanie, ponieważ większość zasobów nie była w pełni wykorzystywana,
+a utrzymanie wielu fizycznych maszyn było kosztowne.
+
+**Era wdrożeń w środowiskach wirtualnych:** Jako rozwiązanie zaproponowano wirtualizację, która umożliwiała
+uruchamianie wielu maszyn wirtualnych (VM) na jednym procesorze fizycznego serwera. Wirtualizacja pozwalała
+izolować aplikacje pomiędzy maszynami wirtualnymi i osiągnąć pewien poziom bezpieczeństwa, jako że informacje
+związane z jedną aplikacją nie były w łatwy sposób dostępne dla pozostałych.
+
+Wirtualizacja pozwala lepiej wykorzystywać zasoby fizycznego serwera i lepiej skalować,
+ponieważ aplikacje mogą być łatwo dodawane oraz aktualizowane, pozwala ograniczyć koszty sprzętu
+oraz ma wiele innych zalet.
+Za pomocą wirtualizacji można udostępnić wybrane zasoby fizyczne jako klaster maszyn wirtualnych "wielokrotnego użytku".
+
+Każda maszyna wirtualna jest pełną maszyną zawierającą własny system operacyjny
+pracujący na zwirtualizowanej warstwie sprzętowej.
+
+**Era wdrożeń w kontenerach:** Kontenery działają w sposób zbliżony do maszyn wirtualnych,
+ale mają mniejszy stopnień wzajemnej izolacji, współdzieląc ten sam system operacyjny.
+Kontenery określane są mianem "lekkich". Podobnie, jak maszyna wirtualna,
+kontener posiada własny system plików, udział w zasobach procesora, pamięć, przestrzeń procesów itd.
+Ponieważ kontenery nie są związane z leżącymi poniżej warstwami infrastruktury,
+mogą być łatwiej przenoszone pomiędzy chmurami i różnymi dystrybucjami systemu operacyjnego.
+
+Kontenery zyskały popularność ze względu na swoje zalety, takie jak:
+
+* Szybkość i elastyczność w tworzeniu i instalacji aplikacji:
+obraz kontenera buduje się łatwiej niż obraz VM.
+* Ułatwienie ciągłego rozwoju, integracji oraz wdrażania aplikacji (*Continuous development,integration, and deployment*):
+obrazy kontenerów mogą być budowane w sposób wiarygodny i częsty.
+Wycofywanie zmian jest skuteczne i szybkie (ponieważ obrazy są niezmienne).
```

Comment:
nieprawda :D

Reply:
Tak brzmi oryginał: "_Continuous development, integration, and deployment: provides for reliable and frequent container image build and deployment with quick and efficient rollbacks (due to image immutability)_". 

Trochę zmieniłem tłumaczenie.